Class/D-functions/exercise1.js

```diff
@@ -8,7 +8,9 @@ function sayHello(){
 // sayHello();
 
 // Add code below so we print "Hello" 3 times
-
+sayHello();
+sayHello();
+sayHello();
 
 

```

Comment:
Good. Don't forget you can use a `for` loop when you want to do something more than once which has the advantage that it's easy to adjust and you don't need to repeat code.